app/actions/remote/scheduled_post.ts

```diff
@@ -2,26 +2,34 @@
 // See LICENSE.txt for license information.
 
 import {forceLogoutIfNecessary} from '@actions/remote/session';
+import {Client} from '@client/rest';
+import DatabaseManager from '@database/manager';
 import NetworkManager from '@managers/network_manager';
-import websocketManager from '@managers/websocket_manager';
 import {getFullErrorMessage} from '@utils/errors';
 import {logError} from '@utils/log';
 
-export async function createScheduledPost(serverUrl: string, schedulePost: ScheduledPost, connectionId?: string, fetchOnly = false) {
-    try {
-        const client = NetworkManager.getClient(serverUrl);
-        const ws = websocketManager.getClient(serverUrl);
+export async function createScheduledPost(serverUrl: string, schedulePost: ScheduledPost, connectionId?: string) {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
+        return {error: `${serverUrl} database not found`};
+    }
 
-        const created = await client.createScheduledPost(schedulePost, ws?.getConnectionId());
+    let client: Client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
 
-        if (!fetchOnly) {
-            // TODO - to be implemented later once DB tables are ready
-            // await operator.handleScheduledPost({scheduledPosts: [created], prepareRecordsOnly: false});
-        }
-        return {scheduledPost: created};
+    let response;
+    try {
+        response = await client.createScheduledPost(schedulePost, connectionId);
     } catch (error) {
         logError('error on createScheduledPost', getFullErrorMessage(error));
         forceLogoutIfNecessary(serverUrl, error);
-        return {error};
+        return {error: getFullErrorMessage(error)};
     }
+
+    // TODO - record scheduled post in database here
+    return {data: true, response};
```

Comment:
Isn't the data the response?

Reply:
@larkox I used the `{data: true}` pattern from existing code. I don't know if it serves any purpose, but its being used like that in too many places so I followed the same pattern.

Reply:
@rahimrahman done! That looks neat.